ports/common/src/iot_network_openssl.c

```diff
@@ -781,6 +794,26 @@ IotNetworkError_t IotNetworkOpenssl_SetReceiveCallback( IotNetworkConnection_t p
 
 /*-----------------------------------------------------------*/
 
+IotNetworkError_t IotNetworkOpenssl_SetCloseCallback( IotNetworkConnection_t pConnection,
+                                                      IotNetworkCloseCallback_t closeCallback,
+                                                      void * pContext )
+{
+    IOT_FUNCTION_ENTRY( IotNetworkError_t, IOT_NETWORK_BAD_PARAMETER );
+
+    if( closeCallback != NULL )
+    {
+        /* Set the callback and parameter. */
+        pConnection->closeCallback = closeCallback;
+        pConnection->pCloseContext = pContext;
+
+        status = IOT_NETWORK_SUCCESS;
+    }
+
+    IOT_FUNCTION_CLEANUP_END();
```

Comment:
Should be `IOT_FUNCTION_EXIT_NO_CLEANUP()` to be consistent with the way we use these macros. Same for the mbedtls file.

Reply:
Fixed